auraed/src/runtime/cells/cell_service.rs

```diff
@@ -54,6 +54,7 @@ impl CellService {
         CellService { cells: Default::default() }
     }
 
+    #[tracing::instrument]
```

Comment:
I assume we want these on the grpc trait impl? Did you put them here because the aurae-photo structs are not `Debug`?

Reply:
no I just put it here as the obvious (to me) entry points.